src/Compilers/CSharp/Portable/Binder/Binder_Patterns.cs

```diff
@@ -113,20 +113,6 @@ internal virtual BoundExpression BindSwitchExpressionCore(SwitchExpressionSyntax
 
         private BoundPattern BindDiscardPattern(DiscardPatternSyntax node, TypeSymbol inputType, bool hasErrors, DiagnosticBag diagnostics)
```

Comment:
>bool hasErrors, DiagnosticBag diagnostics [](start = 97, length = 41)

Not used.

Reply:
The parameters `hasErrors` and `diagnostics` are not used.

---
In reply to: [230157525](https://github.com/dotnet/roslyn/pull/30847#discussion_r230157525) [](ancestors = 230157525)